### DISCUSSION TITLE
it dosen't auto render

### DISCUSSION THREAD

**Discussion by luqiudi** [bcakmakoglu/vue-flow#706] [Q&A]
when i add some edges and nodes, and fit view ,it dosen't render,i must manully drag then it rerender. i dont know why.
do you have a api to manully trigger rerender.

**Comment by bcakmakoglu** [comment]
You have to explain your issue a bit more: What isn't rendering?
Are you getting any errors?
Can you provide some code or a reproduction (sandbox) for the issue?
